website/src/data/components/choropleth/props.js

```diff
@@ -65,6 +65,15 @@ const props = [
         controlType: 'quantizeColors',
         group: 'Style',
     },
+    {
+        key: 'domain',
+        help: 'Defines uppper and lower bounds of color shading',
+        type: 'number[]',
+        required: true,
+        defaultValue: ChoroplethDefaultProps.domain,
+        controlType: 'domainColors',
+        group: 'Style',
```

Comment:
Maybe it's more something for the `Base` group as it relates to the data.

Reply:
Changed to Base group.